src/System.Net.Sockets/tests/FunctionalTests/DisposedSocketTests.netcoreapp.cs

```diff
@@ -0,0 +1,53 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using Xunit;
+
+namespace System.Net.Sockets.Tests
+{
+    public partial class DisposedSocket
+    {
+        //[ActiveIssue(37044, TestPlatforms.AnyUnix)]
```

Comment:
I will uncomment this once I verify via CI that it does indeed fail on all of our tested Unix platforms.